src/middlewares/authorization.ts

```diff
@@ -0,0 +1,35 @@
+import { NextFunction, Request, Response } from "express";
+import { oktaJwtVerifier } from "../lib/okta/connect";
+import { AUTH_SERVER } from "../config";
+import { ForbiddenRequestError, UnauthorizedRequestError } from "../lib/errors";
+
+async function authorizationMiddleware(
+    req: Request,
+    res: Response,
+    next: NextFunction
+): Promise<void> {
+    const { headers } = req;
+    const { authorization = "" } = headers;
+    const token = authorization.match(/Bearer (.+)/) || undefined;
```

Comment:
I'd extract this part in a dedicated function with an interface like `TokenExtractorInterface::extractToken(Request req): Token`